lib/chef/knife/vsphere_datastore_list.rb

```diff
@@ -18,23 +18,6 @@
 require 'chef/knife'
 require 'chef/knife/base_vsphere_command'
 
-def number_to_human_size(number)
```

Comment:
No worries, thanks for taking the time to give feedback it is very much appreciated!

ahh ok that makes sense.